### PR TITLE
Update font sizes and texts on material card and offer footer

### PR DESCRIPTION
**Pull request by dreampulse**

### CODE REVIEW

File: src/app/component/description-list/_description-list.scss [hunk 1]
```diff
@@ -3,14 +3,14 @@
   grid-template-columns: auto 1fr;
   grid-column-gap: $space-l;
   align-items: baseline;
-  font-size: $font-size-s;
+  font-size: $font-size-base;
```
Comment (webholics) [inline src/app/component/description-list/_description-list.scss]: the changes here also affect the offer list is this expected?

Comment (dreampulse) [reply src/app/component/description-list/_description-list.scss]: nope...

File: src/app/component/offer-footer/_offer-footer.scss [hunk 2]
```diff
@@ -2,5 +2,15 @@
   border-top: 2px solid $color-bg-minor;
   background: $color-bg;
   padding: $space-base $space-xl;
-  font-size: $font-size-s;
+  font-size: $font-size-xs;
+
+  > .DescriptionList {
+    font-size: $font-size-s;
+
+    & > dt {
```
Comment (dreampulse) [inline src/app/component/offer-footer/_offer-footer.scss]: As far as I know we want to refactor the footer anyways. So I think this is fine for now

Comment (webholics) [reply src/app/component/offer-footer/_offer-footer.scss]: we are gonna refactor the footer but this part will surely stay. so i really don't get why it should be ok to do hacks like this.

File: src/app/component/offer-footer/_offer-footer.scss [hunk 3]
```diff
@@ -3,4 +3,30 @@
...
+    font-size: $font-size-l;
+    margin-left: $space-l;
+    vertical-align: baseline;
+  }
+
+  &__subline {
+    font-size: $font-size-xs;
+  }
+
+  > .DescriptionList {
```
Comment (ranzwertig) [inline src/app/component/offer-footer/_offer-footer.scss]:
```suggestion
 &  > .DescriptionList {
```

**Pull request merged**